### PR TITLE
Upgrade notes for AWS Security Lake input parsing adjustments

### PR DESCRIPTION
**Pull request by danotorrey**
Update the `UPGRADING.md` file to cover parsing changes made in https://github.com/Graylog2/graylog-plugin-enterprise/pull/6247.

/nocl Change log present in above-linked source PR.

### CODE REVIEW

File: UPGRADING.md [hunk 1]
```diff
@@ -28,6 +28,22 @@ to better align with the GIM schema and allow for targeted handling of AD SIDs.
...
+
+Changed fields:
+- `message`: Now contains the full JSON content of the log message. The `vendor_event_description` field now contains the previous `message` field value for backwards-compatibility. The full message can still also optionally be stored in the `full_message` field if the `Store full message` option is enabled.
+- The message `timestamp` field is now set to the current Graylog system date/time, instead of the previously used log `time` value. The `event_created` field now contains the previous `time` value for backwards-compatibility.
+
+Added fields:
+`vendor_event_description`: Contains the value which was previously present in the `message` log field.
+`event_created`: Contains the `time` log value.
+`event_source_input`: Contains the static value `aws_security_lake`.
+`vendor_version`: Contains the `metadata.product.versoin` log value.
```
Comment (kingzacko1) [inline UPGRADING.md:43]:
```suggestion
`vendor_version`: Contains the `metadata.product.version` log value.
```

**Comment by danotorrey** [comment]
Thanks a lot for the feedback and updates @kingzacko1 @ryan-carroll-graylog! This all looks good to me. 

I made a couple more small tweaks: Alphabetized the added fields, and also removed the `full_message` (that it's still there) description. I had added that thinking it would be helpful, but now with all the inputs in there, it seems to distract a bit and is possibly not that helpful. Let me know if y'all want to keep it in there, and we can add it back.

File: UPGRADING.md [hunk 2]
```diff
@@ -34,6 +34,63 @@ to better align with the GIM schema and allow for targeted handling of AD SIDs.
...
+
+Changed fields:
+- `message`: Now contains the full JSON content of the log message. The `vendor_event_description` field now contains the previous `message` field value for backwards-compatibility. 
+- The message `timestamp` field is now set to the current Graylog system date/time, instead of the previously used log `time` value. The `event_created` field now contains the previous `time` value for backwards-compatibility.
+
+Added fields:
+`event_created`: Contains the `time` log value.
+`event_source_input`: Contains the static value `aws_security_lake`.
+`vendor_event_description`: Contains the value which was previously present in the `message` log field.
+`vendor_version`: Contains the `metadata.product.version` log value.
```
Comment (ryan-carroll-graylog) [inline UPGRADING.md:52]: Nit: Noticed missing bullets here where we have them for the other sections @danotorrey 
```suggestion
- `event_created`: Contains the `time` log value.
- `event_source_input`: Contains the static value `aws_security_lake`.
- `vendor_event_description`: Contains the value which was previously present in the `message` log field.
- `vendor_version`: Contains the `metadata.product.version` log value.
```

Comment (danotorrey) [reply UPGRADING.md]: Thanks a lot for seeing this! Fixed.

**Comment by danotorrey** [comment]
If this looks good in it's latest form to you @ryan-carroll-graylog @kingzacko1 then I will go ahead and merge this after the high five.

**Review by kingzacko1 — APPROVED** [review]

**Pull request merged**